xml/System.Net.NetworkInformation/Ping.xml

```diff
@@ -64,7 +64,9 @@
  Network topology can determine whether <xref:System.Net.NetworkInformation.Ping> can successfully contact a remote host. The presence and configuration of proxies, network address translation (NAT) equipment, or firewalls can prevent <xref:System.Net.NetworkInformation.Ping> from succeeding. A successful <xref:System.Net.NetworkInformation.Ping> indicates only that the remote host can be reached on the network; the presence of higher level services (such as a Web server) on the remote host is not guaranteed.  
   
  This class provides functionality similar to the Ping.exe command line tool. The <xref:System.Net.NetworkInformation.Ping.Send%2A> and <xref:System.Net.NetworkInformation.Ping.SendAsync%2A> methods send an Internet Control Message Protocol (ICMP) echo request message to a remote computer and waits for an ICMP echo reply message from that computer. For a detailed description of ICMP messages, see RFC 792, available at [https://www.ietf.org](https://www.ietf.org/).  
-  
+ 
+ The default timeout when timeout is not specified is 5 seconds similar to Ping.exe command line tool timeout.
```

Comment:
Could you please link a source of this information? Is it also true on *nixes?

Reply:
I see no reason to add reference to ping.exe for public API doc. stating the default should be sufficient IMHO.

Reply:
I didn't mean link it here, I was curious where the 5 seconds came from.